sw/device/lib/dif/dif_uart.c

```diff
@@ -124,12 +125,19 @@ dif_result_t dif_uart_configure(const dif_uart_t *uart,
     return kDifBadArg;
   }
 
+  // Check requested RXBLVL is within bounds.
+  uint32_t rxblvl = config.rx_break_level;
+  if ((rxblvl & UART_CTRL_RXBLVL_MASK) != rxblvl) {
```

Comment:
Would you mind testing this branch on the unittest. Also adding a new test for the function below.